src/js/angular/core/directives/dynamic-configuration-form/dynamic-configuration-form.directive.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Enum for configuration types.
+ * @readonly
+ * @enum {string}
+ */
+export const CONFIGURATION_TYPE = {
+    STRING: 'string',
+    TEXT: 'text',
```

Comment:
What is the difference between string and text? I can assume that text should be rendered in textarea but I see no difference between these two

Reply:
fixed